src/components/pages/Landing/RenderLandingPage.js

```diff
@@ -91,10 +92,12 @@ function RenderLandingPage(props) {
               school districts.
             </p>
           </div>
-          <Button
-            classType="landing-second-container-btn"
-            buttonText="VISIT WEBSITE"
-          ></Button>
+          <a href="https://codeyourdreams.org">
```

Comment:
Same as above 

Since we are using React Router, I think we should use the Link component instead of the a tag. The Link component is able to do navigation without forcing the browser to refresh.

Reply:
We have implemented the changes and made them all links.